CanalTP/NmmPortalBundle
# List customer without status locked

# Description

This PR filter customer on the locked status

## Issue

Issue link: BOT-1179, BOT-1151

## How to test

Here are the following steps to test this pull request:

- change a customer settings on cus_locked field to True (database)
- connect into admin nmm on the customer list
- you should not see the customer that you have changed
